src/d3a/models/area/stats.py

```diff
@@ -172,20 +169,3 @@ def get_market_stats(self, market_slot_list, dso=False):
                     self._get_market_area_throughput(time_slot)
 
         return out_dict
-
-    def aggregate_exported_imported_energy(self, area):
```

Comment:
Please have a look at `_get_market_area_throughput` . The DSO stats need the exported and imported energy. This is why i put this function here into the stats.py.

Reply:
Thanks a lot for pointing, got overlooked by me, will come-up  with a solution.